reward_endpoint.py

```diff
@@ -9,13 +9,14 @@
 from web_utils import auth_request_get_params, bad_request, auth_request, auth_request_get_single_param
 import utils
 import email_utils
-from app_core import app, db, limiter
+from app_core import app, db, limiter, csrf
 from models import User, Role, Referral
 
 logger = logging.getLogger(__name__)
 reward = Blueprint('reward', __name__, template_folder='templates')
 limiter.limit("100/minute")(reward)
 use_referrals = app.config["USE_REFERRALS"]
+csrf.exempt(reward)
```

Comment:
move this one line up so it is with the other blueprint modifying statements

Reply:
this has been moved. resolving.